base/bucket_gocb.go

```diff
@@ -61,8 +61,13 @@ type CouchbaseBucketGoCB struct {
 
 type GoCBLogger struct{}
 
-func (l GoCBLogger) Output(s string) error {
-	LogTo("gocb", s)
+func (l GoCBLogger) Log(level gocbcore.LogLevel, offset int, format string, v ...interface{}) error {
+	switch level {
+	case gocbcore.LogError:
+		LogError(fmt.Errorf(format, v))
+	default:
```

Comment:
I was expecting to also use Warn for gocbcore.LogWarn - let me know if you left that out intentionally.